debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunnerUtils.kt

```diff
@@ -32,17 +31,15 @@ object RsDebugRunnerUtils {
         state: CargoRunStateBase,
         environment: ExecutionEnvironment,
         runExecutable: GeneralCommandLine
-    ): RunContentDescriptor? {
+    ): RunContentDescriptor {
         val runParameters = RsDebugRunParameters(environment.project, runExecutable, state.cargoProject)
         return XDebuggerManager.getInstance(environment.project)
-            .startSession(environment, object : XDebugProcessConfiguratorStarter() {
```

Comment:
> For some reason, Rider debugging integration fails when the platform calls `XDebugProcessConfiguratorStarter#configure`

Sounds strange to me, I hope `XDebugProcessStarter` really solves this and there are no other unexpected problems